### PR TITLE
Add r-robustgasp

### PR DESCRIPTION
**Pull request by thealanjason**
I have used `conda_r_skeleton_helper ` to create this recipe.
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-robustgasp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/RobustGaSP_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/RobustGaSP/RobustGaSP_{{ version }}.tar.gz
+  sha256: 0ed297111afd3aaf0d34e8ef5e48e2f5476f6be04a53b0fc2711efd5c707dc9a
+
+build:
+  merge_build_host: True  # [win]
```
Comment (mfansler) [inline recipes/r-robustgasp/meta.yaml:16]: Workaround for missing DSO:

```suggestion
  merge_build_host: True  # [win]
  missing_dso_whitelist:
    - '*/R.dll'           # [win]
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Mostly good; just need this workaround for R.dll on Windows.

**Comment by thealanjason** [comment]
Out of curiosity, does conda build automatically for osx-arm64 and linux-arm64?

File: recipes/r-robustgasp/meta.yaml (hunk 1 above)
Comment (thealanjason) [reply recipes/r-robustgasp/meta.yaml:16]: Thanks for the prompt response. 
I’d like to know what a DSO is? Would appreciate any info to read on :)

**Comment by mfansler** [comment]
> Out of curiosity, does conda build automatically for osx-arm64 and linux-arm64?

No. Those platforms must be added after feedstock creation.

**Comment by thealanjason** [comment]
> No. Those platforms must be added after feedstock creation.

Since I'm trying this for the first time, again, are there any resources where I can read about this and learn how this is done?

Also, how do I get somebody from `conda-forge/r` to comment on this PR?

**Comment by mfansler** [comment]
> > No. Those platforms must be added after feedstock creation.
> 
> Since I'm trying this for the first time, again, are there any resources where I can read about this and learn how this is done?

Absolutely. See the following sections in the Maintainer documentation:

 - [*"Apple Silicon builds"*](https://conda-forge.org/docs/maintainer/knowledge_base.html#apple-silicon-builds)
 - [*"Using `arch_rebuild.txt`"*](https://conda-forge.org/docs/maintainer/knowledge_base.html#using-arch-rebuild-txt)

> Also, how do I get somebody from `conda-forge/r` to comment on this PR?

I'm on the team 😉 That lint can be safely ignored - it's [a bug in the linter](https://github.com/conda-forge/conda-smithy/issues/1817).

**Review by mfansler — APPROVED** [review]

**Comment by mfansler** [comment]
Thanks for submitting this! ❤️

**Pull request merged**